src/components/v2/V2Project/V2ProjectReconfigureModal/V2ReconfigureUpcomingMessage.tsx

```diff
@@ -0,0 +1,64 @@
+import { Trans } from '@lingui/macro'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import { useContext } from 'react'
+import { BigNumber } from '@ethersproject/bignumber'
+
+import { detailedTimeString, secondsUntil } from 'utils/formatTime'
+
+import { getBallotStrategyByAddress } from 'constants/v2/ballotStrategies/getBallotStrategiesByAddress'
+
+export default function V2ReconfigureUpcomingMessage() {
+  const { fundingCycle } = useContext(V2ProjectContext)
+  if (!fundingCycle) return null
+
+  const currentFCNumber = fundingCycle.number.toNumber()
+
+  const ballotStrategy = getBallotStrategyByAddress(fundingCycle.ballot)
+  const ballotStrategyLength = ballotStrategy.length
+
+  const duration = fundingCycle.duration.toNumber()
+
+  const secondsUntilNextFC = secondsUntil(fundingCycle.start.add(duration))
+
+  // Separating the full message out like this for translation purposes
+  // (they need full sentences, can't chop and change)
+  if (!duration || duration === 0) {
+    // If duration is unset/0, changes take effect immediately to current FC
+    return (
+      <Trans>
+        Because you do not have a funding cycle duration, changes you make will
```

Comment:
```suggestion
        You don't have a funding cycle duration. Changes you make will
```

Common contractions like "don't" are fine in UI copy :)